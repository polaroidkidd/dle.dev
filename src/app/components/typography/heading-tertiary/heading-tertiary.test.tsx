import React from "react";
import renderer from "react-test-renderer";
import { HeadingTertiary } from "./index";

test("Render HeadingTertiary with text", () => {
  const tree = renderer.create(<HeadingTertiary text="test heading"/>).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Render HeadingTertiary with empty string", () => {
  const tree = renderer.create(<HeadingTertiary text=""/>).toJSON();
  expect(tree).toMatchSnapshot();
});
