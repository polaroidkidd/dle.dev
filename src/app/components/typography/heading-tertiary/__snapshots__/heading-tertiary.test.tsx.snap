// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HeadingTertiary with empty string 1`] = `
<h3
  className="heading-tertiary"
>
  
</h3>
`;

exports[`Render HeadingTertiary with text 1`] = `
<h3
  className="heading-tertiary"
>
  test heading
</h3>
`;
